### PR TITLE
Publish Voyager 5.0.0-rc.11

### PR DESCRIPTION
**Pull request by tamalsaha**

### CODE REVIEW

File: stable/voyager/values.yaml [hunk 1]
```diff
@@ -3,32 +3,32 @@
...
+  imageTag: 5.0.0-rc.11
 ## Optionally specify an array of imagePullSecrets.
 ## Secrets must be manually created in the namespace.
 ## ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
 ##
 # imagePullSecrets:
 #   - name: myRegistryKeySecretName
 ## Specify a imagePullPolicy
-## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+## ref: http://kubernetes.io/docs/guides/images/#pre-pulling-images
```
Comment (sameersbn) [inline stable/voyager/values.yaml]: this change inserts a broken link

File: stable/voyager/values.yaml [hunk 2]
```diff
@@ -3,32 +3,32 @@
...
 imagePullPolicy: IfNotPresent
-## Use cloud provider here. Read details https://github.com/appscode/voyager/blob/master/docs/user-guide/README.md
+## Use cloud provider here.
 cloudProvider:
 ## The path to the cloud provider configuration file. Empty string for no configuration file.
 ## ie. for azure use /etc/kubernetes/azure.json
 cloudConfig: ''
 ## Installs voyager operator as critical addon
-## https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
+## https://kubernetes.io/docs/guides/administer-cluster/guaranteed-scheduling-critical-addon-pods/
```
Comment (sameersbn) [inline stable/voyager/values.yaml]: broken link. i believe this change was inadvertent ?

File: stable/voyager/values.yaml [hunk 3]
```diff
@@ -37,7 +37,7 @@ persistence:
   hostPath: /etc/kubernetes
 
 ## Node labels for pod assignment
-## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+## Ref: https://kubernetes.io/docs/guides/node-selection/
```
Comment (sameersbn) [inline stable/voyager/values.yaml]: same as above.

**Review by sameersbn — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
@sameersbn, fixed the links.

**Comment by tamalsaha** [comment]
Thanks @sameersbn ! Fixed the links.

**Review by sameersbn — APPROVED** [review]

**Pull request merged**